src/libs/Pusher/pusher.ts

```diff
@@ -226,7 +226,7 @@ function subscribe(channelName, eventName, eventCallback = () => {}, onResubscri
                 onResubscribe();
             });
 
-            channel.bind('pusher:subscription_error', (data = {}) => {
+            channel.bind('pusher:subscription_error', (data: {type?: string; error?: string; status?: string} = {}) => {
```

Comment:
Could you extract `{type?: string; error?: string; status?: string}` into a separate type?